### PR TITLE
Update install/run instructions for macOS

### PR DESCRIPTION
**Pull request by rudolfs**
Refs: https://github.com/radicle-dev/radicle-upstream/pull/1406.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -51,9 +51,13 @@ follow along development in our [Matrix channel][mc] or on
...
 Radicle Upstream is available for download on Linux and macOS (version 10.14 and
-newer). 
+newer).
 
-**-> [Download Radicle Upstream][bk]**
+**→ [Download Radicle Upstream for Linux (AppImage)][d1]**
+
+**→ [Download Radicle Upstream for Linux (snap)][d2]**
+
+**→ [Download Radicle Upstream for macOS (dmg)][d3]**
```
Comment (abbey-titcomb) [inline docs/getting-started.md:60]: Nice 👌

**Review by abbey-titcomb — APPROVED** [review]
Looks great to me! Thanks for cleaning up

**Review by xla — APPROVED** [review]
👱 🎤 🗯 😅

**Review by abbey-titcomb — APPROVED** [review]
Loooooks greaaat

**Pull request merged**